operator/config/samples/csiscaleoperators.csi.ibm.com_cr.yaml

```diff
@@ -59,7 +59,7 @@ spec:
 
 # Liveness probe image name, in case we do not want to use default image.
 # ==================================================================================
-#  livenessprobe: "us.gcr.io/k8s-artifacts-prod/sig-storage/livenessprobe:v2.3.0"
+#  livenessprobe: "us.gcr.io/k8s-artifacts-prod/sig-storage/livenessprobe:v2.3.1"
```

Comment:
This is not valid. Liveness probe version we have to keep 2.3.0 only

Reply:
fixed.